loader.js

```diff
@@ -36,89 +36,84 @@ module.exports.pitch = function(request) {
 			resultSource = undefined;
 		}
 
-		if(query.extract !== false) {
```

Comment:
Lesson learned next time I'll split it into two commits (delete and reformat) so git doesn't get confused.